content/en/blog/2025/sail-operator-ga/index.md

```diff
@@ -0,0 +1,302 @@
+
+## Try it out
+
+Do you want to try our operator?
+Going over this example will show you how to safely do an update of your istio control plane by using the revision-based upgrade strategy, this means you will have two Istio control planes running at the same time,allowing you to migrate workloads easily, minimizing the risk of traffic disruptions.
+
+Prerequisites:
+- Running cluster
+- Helm
+- Kubectl
+- Istioctl
+
+### Install the operator from the sail operator helm repository
+
+{{< text bash >}}
+$ helm repo add sail-operator https://istio-ecosystem.github.io/sail-operator
+$ helm repo update
+$ kubectl create namespace sail-operator
+$ helm install sail-operator sail-operator/sail-operator --version 1.0.0 -n sail-operator
+{{< /text >}}
+
+The operator is now installed in your cluster:
+
+{{< text plain >}}
+NAME: sail-operator
+LAST DEPLOYED: Tue Mar 18 12:00:46 2025
+NAMESPACE: sail-operator
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+{{< /text >}}
+
+### Check the operator pod running
+
+{{< text bash >}}
+$ kubectl get pods -n sail-operator
+{{< /text >}}
+
+{{< text plain >}}
+NAME                             READY   STATUS    RESTARTS   AGE
+sail-operator-56bf994f49-j67ft   1/1     Running   0          87s
+{{< /text >}}
+
+#### Create an Istio resource with istio version 1.24.2 and a Istio revision tag
+
+{{< text bash >}}
+$ kubectl create ns istio-system
+$ cat <<EOF | kubectl apply -f-
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+  name: default
+spec:
+  namespace: istio-system
+  updateStrategy:
+    type: RevisionBased
+    inactiveRevisionDeletionGracePeriodSeconds: 30
+  version: v1.24.2
+---
+apiVersion: sailoperator.io/v1
+kind: IstioRevisionTag
+metadata:
+  name: default
+spec:
+  targetRef:
+    kind: Istio
+    name: default
+EOF
+{{< /text >}}
+
+Note that `IstioRevisionTag` has target reference to `Istio` resources with name default.
+
+Check the state of the resources created:
+- `istiod` pods are running
+
+{{< text bash >}}
+$ kubectl get pods -n istio-system
+{{< /text >}}
+
+{{< text plain >}}
+NAME                                    READY   STATUS    RESTARTS   AGE
+istiod-default-v1-24-2-bd8458c4-jl8zm   1/1     Running   0          3m45s
+{{< /text >}}
+
+- `Istio` resource created
+
+{{< text bash >}}
+$ kubectl get istio
+{{< /text >}}
+
+{{< text plain >}}
+NAME      REVISIONS   READY   IN USE   ACTIVE REVISION   STATUS    VERSION   AGE
+default   1           1       1        default-v1-24-2   Healthy   v1.24.2   4m27s
+{{< /text >}}
+
+- `IstioRevisionTag` resource created
+
+{{< text bash >}}
+$ kubectl get istiorevisiontag
+{{< /text >}}
+
+{{< text plain >}}
+NAME      STATUS                    IN USE   REVISION          AGE
+default   NotReferencedByAnything   False    default-v1-24-2   4m43s
+{{< /text >}}
+
+Note that the `IstioRevisionTag` status is `NotReferencedByAnything`, this is because there is no resource referenced using the revision `default-v1-24-2`.
```

Comment:
```suggestion
Note that the `IstioRevisionTag` status is `NotReferencedByAnything`, this is because there are currently no resources using the revision `default-v1-24-2`.
```